README.md

```diff
@@ -40,7 +40,7 @@ bitflags! {
         const A = 0b00000001;
         const B = 0b00000010;
         const C = 0b00000100;
-        const ABC = Self::A.bits | Self::B.bits | Self::C.bits;
+        const ABC = Self::A.bits() | Self::B.bits() | Self::C.bits();
```

Comment:
Why do a function call when the value is there in a variable?

Reply:
In `2.x` it's no longer in a field called `bits`, the library generates a newtype so the quickest migration is to use the method call instead.